tests/test.js

```diff
@@ -13,17 +13,26 @@ var c = new GoogleContacts({
 c.getContacts(function (err, contacts) {
   if (err) throw err;
   assert.ok(typeof contacts === 'object', 'Contacts is not an object');
-  console.log(contacts);
+  console.log('Showing first 10 contacts out of '+ contacts.length)
+  console.log(contacts.slice(0,9));
+  // setting first contact as the id for the getContact test
   contactsTested = true;
-});
+  test_id = contacts[0].id
 
-c.getContact(function (err, contact) {
-  if (err) throw err;
-  assert.ok(typeof contact === 'object', 'Contact is not an object');
-  console.log(contact);
-  contactTested = true;
+  // this one needs a valid id
+  c.getContact(function (err, contact) {
+    if (err) throw err;
+    assert.ok(typeof contact === 'object', 'Contact is not an object');
+    console.log('Showing individual contact (first from previous test)')
+    console.log(contact);
+    contactTested = true;
+  }, {
+    id: test_id
+  });
 });
```

Comment:
Result of `c.getContact()`

![image](https://cloud.githubusercontent.com/assets/211358/21242356/da497848-c30b-11e6-8c18-5edf1f0db76d.png)